dcap-sp1-cli/src/constants.rs

```diff
@@ -0,0 +1,12 @@
+// Collateral Path Defaults
+pub const DEFAULT_QUOTE_PATH: &str = "../data/quote.hex";
+
+// Chain Defaults
+pub const DEFAULT_RPC_URL: &str = "https://automata-testnet.alt.technology";
```

Comment:
Change it to `https://rpc-testnet.ata.network` or `https://1rpc.io/ata/testnet`.